### PR TITLE
Fix: Display IconCard's copy alert Over Navbar

### LINKED ISSUE #4729: "Copied icon to Clipboard" dialog box gets partially hidden under the navbar
### Reproduce The Error

- Go to - https://linkfree.eddiehub.io/icons
- Search for an icon and then click on the IconCard
- The "Copied `icon` to Clipboard" dialog box gets hidden under the navbar.

### Issue

https://user-images.githubusercontent.com/116095538/218293677-478ab8f8-a9b5-450f-95e3-036fb74f046d.mp4

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by GauravBurande**
## Fixes Issue #4729

fixes #4729 

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4743"></a>

### CODE REVIEW

File: components/navbar/Navbar.js [hunk 1]
```diff
@@ -172,11 +172,10 @@ export default function Navbar() {
         </div>
 
         <div
-          className={`${
-            isOpen
+          className={`${isOpen
               ? "transform translate-y-0 opacity-100"
               : "transform -translate-y-96 opacity-0 "
-          } md:hidden z-20 absolute t-0 bg-gray-800 transition-all duration-700 ease-in-out  w-full`}
+            } md:hidden z-20 absolute t-0 bg-gray-800 transition-all duration-700 ease-in-out  w-full`}
```
Comment (krshkun) [inline components/navbar/Navbar.js:178]: if thhis is a formatting change, this should be another PR I think

File: components/navbar/Navbar.js [hunk 2]
```diff
@@ -88,7 +88,7 @@ export default function Navbar() {
   return (
     <div className="min-h-full" ref={navConRef}>
       <nav className=" relative top-0">
-        <div className=" z-30 bg-gray-800 w-full mx-auto px-4 sm:px-6 lg:px-8 relative t-0">
+        <div className="z-30 bg-gray-800 w-full mx-auto px-4 sm:px-6 lg:px-8 relative t-0">
```
Comment (krshkun) [inline components/navbar/Navbar.js:91]: same with this one

File: components/IconCard.js [hunk 3]
```diff
@@ -24,7 +24,7 @@ export default function IconCard({ iconName }) {
       </button>
       <div
         aria-live="assertive"
-        className="pointer-events-none fixed inset-0 flex items-end px-4 py-6 sm:items-start sm:p-6"
+        className="pointer-events-none fixed inset-0 z-40 flex items-end px-4 py-6 sm:items-start sm:p-6"
```
Comment (krshkun) [inline components/IconCard.js:27]: Wouldn't now the text overlap the navbar?

Comment (GauravBurande) [reply components/IconCard.js:27]: No it's a alert it goes away after 1.5sec

File: components/navbar/Navbar.js (hunk 1 above)
Comment (GauravBurande) [reply components/navbar/Navbar.js:178]: I think the formatter made this changes

File: components/IconCard.js (hunk 3 above)
Comment (GauravBurande) [reply components/IconCard.js:27]: Actually there was no issue, but in recent code changes, I think someone gave navbar zindex of 30 to solve another issue, which caused the alert component display under the navbar, that's my understanding of the current issue.

**Comment by swagataroy30** [comment]
Hey I did check, it fixes the issue but as i see it's covering the the version no. and Login. But i have a suggestion what if we keep the positioning of the dialog box at the same place as we do in mobile view. Here's what i am suggesting and i would also like to know your views on it 

https://user-images.githubusercontent.com/116095538/218313143-563f67a6-3ebf-419b-b06c-f440a497c913.mp4

**Review by swagataroy30 — COMMENTED** [review]
As I see there are no changes made into the Navbar.js file related to this issue. There's need to commit this file.

**Comment by GauravBurande** [comment]
> Hey I did check, it fixes the issue but as i see it's covering the the version no. and Login. But i have a suggestion what if we keep the positioning of the dialog box at the same place as we do in mobile view. Here's what i am suggesting and i would also like to know your views on it
> 
>  2023-02-12.10-12-09.mp4

It looks good, let's see what others think.
